### PR TITLE
'Improved' Data

### PR DESCRIPTION
**Pull request by gordonkelly26**
All link tables filled for 6 complete job families and ft. Chris' removal of silly things. Necessary changes made to create-database script.

### CODE REVIEW

File: Database-Files/CreateDatabase.sql [hunk 1]
```diff
@@ -83,7 +83,6 @@ CREATE TABLE comp_Band (
     compID int NOT NULL,
     bandID int NOT NULL,
     compDesc varchar(200) NOT NULL,
-    PRIMARY KEY(compID, bandID),
```
Comment (ChrisThompsonK) [inline Database-Files/CreateDatabase.sql:86]: Why are we removing these

File: Database-Files/dummy.sql [hunk 2]
```diff
@@ -1,393 +1,550 @@
...
-insert into resp_Role(respID, roleID) values (5, 5);
-insert into resp_Role(respID, roleID) values (6, 6);
-insert into resp_Role(respID, roleID) values (7, 7);
-insert into resp_Role(respID, roleID) values (8, 8);
-insert into resp_Role(respID, roleID) values (9, 9);
-insert into resp_Role(respID, roleID) values (1, 10);
-insert into resp_Role(respID, roleID) values (2, 11);
-insert into resp_Role(respID, roleID) values (3, 12);
-insert into resp_Role(respID, roleID) values (4, 13);
-insert into resp_Role(respID, roleID) values (5, 14);
```
Comment (ChrisThompsonK) [inline Database-Files/dummy.sql:393]: What's happening with these fields

File: Database-Files/dummy.sql [hunk 3]
```diff
@@ -1,393 +1,550 @@
...
-insert into responsibilities (respDesc)
-values ('Encourage collaboration across BUs');
-insert into responsibilities (respDesc)
-values ('Identify and mentor high-potential, junior talent and expedite their personal development');
-insert into responsibilities (respDesc)
-values ('Represent Kainos at careers fairs or Kainos open evenings events if invited');
-insert into responsibilities (respDesc)
-values ('Deliver your tasks within the timelines while adhering to the Kainos quality standards');
-insert into responsibilities (respDesc)
-values ('Immediately tell your manager if your tasks are not going to be complete within the expected timeframe');
```
Comment (ChrisThompsonK) [inline Database-Files/dummy.sql]: Where are we entering these now

File: Database-Files/CreateDatabase.sql [hunk 4]
```diff
@@ -7,7 +7,7 @@ CREATE TABLE training (
     ID int PRIMARY KEY AUTO_INCREMENT,
     name VARCHAR(50) NOT NULL,
     description VARCHAR(200) NOT NULL,
-    trainingType VARCHAR(100)
+    trainingType VARCHAR(200)
 );
```
Comment (Conormc98) [inline Database-Files/CreateDatabase.sql:12]: Training should have a trainingLink field now

`trainingLink VARCHAR(200)`

File: Database-Files/CreateDatabase.sql (hunk 1 above)
Comment (micalexis) [reply Database-Files/CreateDatabase.sql:86]: address these comments!

**Review by micalexis — CHANGES_REQUESTED** [review]
address the comments from you colleagues

File: Database-Files/CreateDatabase.sql (hunk 1 above)
Comment (gordonkelly26) [reply Database-Files/CreateDatabase.sql:86]: It was preventing us inserting id's into the field and an unnecessary constraint.

File: Database-Files/dummy.sql (hunk 2 above)
Comment (gordonkelly26) [reply Database-Files/dummy.sql:393]: I've changed the structure so it's easier to read and update.

File: Database-Files/dummy.sql (hunk 3 above)
Comment (gordonkelly26) [reply Database-Files/dummy.sql]: Same as above, the responsibilities/comp are entered for each band as it is created etc

File: Database-Files/CreateDatabase.sql (hunk 4 above)
Comment (gordonkelly26) [reply Database-Files/CreateDatabase.sql:12]: Will be added now with a link to general sharepoint

**Review by Conormc98 — APPROVED** [review]

**Review by micalexis — APPROVED** [review]

**Review by ChrisThompsonK — APPROVED** [review]
I like it

**Pull request merged**